cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
I think putting it inside `projects/OpenCV/hunter.cmake` with an explicit test on `HUNTER_OpenCV_VERSION` would be the best solution then, as pointed out in the Qt example.

I'll drop a message when I update the code and we can leave this pull request on hold until the change of using Qt from cache and the feedback of @headupinclouds